ui/components/component-library/button-icon/button-icon.js

```diff
@@ -84,14 +85,14 @@ ButtonIcon.propTypes = {
   /**
    * The name of the icon to display. Should be one of ICON_NAMES
    */
-  icon: PropTypes.string.isRequired, // Can't set PropTypes.oneOf(ICON_NAMES) because ICON_NAMES is an environment variable
+  iconName: PropTypes.oneOf(ICON_NAMES).isRequired,
```

Comment:
This is causing a console error during certain e2e tests. I saw it locally with the test `Swaps - notifications -> AssertionError [ERR_ASSERTION]: Expected values to be strictly equal`.

The error is:

```
Warning: Invalid argument supplied to oneOf, expected an array
```

`ICON_NAMES` is an object here, a mapping of snake-case icon names to kebab-case. I'm guessing you meant to something like this here:

```
iconName: PropTypes.oneOf(Object.values(ICON_NAMES)).isRequired,
```